### ISSUE TITLE
Doesn't work on GitLab

### ISSUE THREAD

**Issue by fis2009** [mockery/mockery#1185]
Hello everyone,
I Use Mockery within one of my projects, - this is the first time I use it -.
I have to CI/CD test my project.
But I always get a message saying that "Mockery is not found ".
I hope that someone will help me solve this problem.

Thanks in advanced.

**Comment by ghostwriter** [comment]
Hey @fis2009,

To better understand the issues and help you solve them, can you please share where you're seeing the "Mockery is not found " error message?

Did you use composer to install Mockery? eg. `composer require --dev mockery/mockery`


P.S. This repo is for **Mockery for PHP**, not **Mockery for Golang**, in case there was a mixup.

**Comment by FirasSahli2010** [comment]
Yes sure, this works on my local computer with windows 11 good but not on CI/CD repository
